.lintr_new

```diff
@@ -25,6 +29,7 @@ linters: linters_with_defaults(
     strings_as_factors_linter(),
     unnecessary_lambda_linter(),
     unneeded_concatenation_linter(allow_single_expression = FALSE),
+    unused_import_linter(),
```

Comment:
Why is that useful to us as a package? It only lints `library()` and friends.

Reply:
It's useful for us to make sure that contributors don't include these lints in their contributions, and the CI/CD catches them even if the reviewers miss it for some reason.

Reply:
I expect `tests/testthat.R` to produce these lints because it loads some packages without actively using it. Can you confirm this?